promscale/index.md

```diff
@@ -0,0 +1,10 @@
+# Promscale
```

Comment:
can we use the text from github (below). I see the diff between this text and the website as:
- no architecture diagram
- not pointing users to our slack channel
- no mention of the design doc
- no mention of the compliance suite 
```
Promscale is an open-source long-term store for Prometheus data designed for
analytics. It is a **horizontally scalable**, **highly-compressed**, and
**operationally mature** platform for Prometheus data that offers the
combined power of PromQL and SQL, enabling developers to ask any question,
create any dashboard, and achieve greater visibility into their systems.
Promscale is built on top of [TimescaleDB](https://www.timescale.com/), the
leading relational database for time-series.


<img src="docs/assets/promscale-arch.png" alt="Promscale Architecture Diagram" width="800"/>

By allowing a user to use SQL, in addition to
PromQL, this platform empowers the user to ask complex analytical
queries from their metrics data, and thus extract more meaningful
insights.

✅  Promscale has consistently been one of the only long-term stores for Prometheus data that continue to maintain its top-performance, receiving
[100% compliance test score](https://promlabs.com/promql-compliance-test-results/2020-12-01/promscale)
each time (with no cross-cutting concerns) from PromLab's
[PromQL Compliance Test Suite](https://promlabs.com/promql-compliance-tests/).

For a detailed description of this architecture, please see [our design
doc][design-doc].

If you have any questions, please join the #promscale channel on
[TimescaleDB Slack](https://slack.timescale.com/).
```